.github/workflows/python_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: python_tests
+on: [push]  # pull_request,
+jobs:
+  python_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.8]  # , 3.9]
+    runs-on: ubuntu-20.04
+    steps:
```

Comment:
SUGGESTION: Can we group these into named steps? Something like:

```yml
- name: Checkout
  uses: actions/checkout@v2
  run: make git
- name: Installs
   run: |
        sudo apt-get install libxml2-dev libxslt-dev
        pip install --upgrade pip setuptools wheel
        pip install -r requirements_test.txt
- name: Lint
 ...
- name: Build
 ...
- name: Test
```

Reply:
The problem with bundling the steps is that when a command fails, you need to hunt to see which one it is.  This can require a lot of scrolling with commands that generate copious logs.  If I do not bundle these, can instantly spot which command failed and can directly study only its logs.

Reply:
The problem with names is that they hide the details.  With names I know only `Install` but without them, I know `pip install -r requirements_test.txt`.  People are not going to look at this stuff when tests pass.  But when tests fail they want to jump straight to the single command that failed and they want to know the exact command that was typed.  It is already one level of indirection that key command details are buried in Makefile but we add a second level of indirection when we say that the problem was in one of the three commands in the Install step.

Reply:
:+1: